### ISSUE TITLE
IP data inversion with ERTIPManager

### ISSUE THREAD

**Issue by HenryWHR** [gimli-org/gimli#765]
Hi,

Hope you all had a nice summer!

I am using ERTIPManager for IP data inversion.
```python
mgr = ert.ERTIPManager(data,verbose=True)
mgr.setMesh(mesh)
mgr.invert(zWeight=0.3, lam=20)
```

I have two question regarding the inversion and the output.
1), How can I define the lambda for IP inversion, similar to 'lam' for DC resistivity.
2), How can I get the response of the final IP model, like 'mgr.inv.response' for resistivity model.

Many thanks for your kind help!

Best regards,
Henry

**Comment by halbmy** [comment]
Dear Henry,
instead of `mgr.invert` which currently uses the same `lam` argument, you should better do it in two calls 
```
mgr.invertDC(lam=...)
mgr.invertTDIP(lam=...)
```

**Comment by halbmy** [comment]
To the second question: I made the inversion object `invIP` available (2eec1c77)

**Comment by HenryWHR** [comment]
Thanks a lot, Thomas! 
Solved!
